app/views/collections/show.html.slim

```diff
@@ -49,22 +65,22 @@
 .row
   .col-md-12
     .actions.btn-group[role="group"]
-      - if @user
+      - if action_name == 'view_shared'
         = link_to t('.button.collaborate'), save_shared_collection_path(@collection), method: 'post', class: 'btn btn-important'
         = link_to t('common.button.back'), user_messages_path(current_user), class: 'btn btn-important'
       - else
-        .btn-group
-          = button_tag type: 'button', 'data-bs-toggle': 'dropdown', aria: {haspopup: "true", expanded: "false"}, class: 'btn btn-important nav-btn-exercise dropdown-toggle split' do
-            => t('.button.share')
-            span.caret
-          .dropdown-menu#share-menu
-            .dropdown-header style="padding-left: 2px;"
-              = t('.type_hint')
-              | :
-            .input-group
-              = text_field_tag 'user', nil, form: 'share', class: 'form-control'
-              = submit_tag t('.button.share'), form: 'share', class: 'btn btn-light', type: 'button'
         - if policy(@collection).edit?
+          .btn-group
+            = button_tag type: 'button', 'data-bs-toggle': 'dropdown', aria: {haspopup: "true", expanded: "false"}, class: 'btn btn-important nav-btn-exercise dropdown-toggle split' do
+              => t('.button.share')
+              span.caret
+            .dropdown-menu#share-menu
+              .dropdown-header style="padding-left: 2px;"
```

Comment:
What is your "policy" regarding inline styles? 🤔

Reply:
We don't really have a policy, but I try to avoid it normally